idp/idpclient/client.go

```diff
@@ -0,0 +1,171 @@
+// name 'idpclient' is used although it repeats the name of the outer package 'idp'
+// in order to improve readability functions like e.g. idpclient.New() or idpclient.HttpClient()
+package idpclient
+
+import (
```

Comment:
Run goimports over imports. (Grouped imports by standard and external packages)

Reply:
Done